--- search_utils.py.orig	2023-04-19 10:15:51.411155006 +0200
+++ search_utils.py	2023-04-19 10:16:38.201782826 +0200
@@ -1170,7 +1170,7 @@
     @classmethod
     def filter(cls, model_versions, filter_string):  # pylint: disable=arguments-renamed
         """Filters a set of model versions based on a search filter string."""
-        model_versions = [mv for mv in model_versions if mv.current_stage != STAGE_DELETED_INTERNAL]
+        model_versions = [mv for mv in model_versions]
         if not filter_string:
             return model_versions
         parsed = cls.parse_search_filter(filter_string)
