--- .venv/lib/python3.10/site-packages/mlflow/store/model_registry/sqlalchemy_store.py	2023-07-14 22:56:12.850158873 +0200
+++ -	2023-07-14 22:58:05.496177568 +0200
@@ -925,7 +925,6 @@
         with self.ManagedSessionMaker() as session:
             query = (
                 filter_query.options(*self._get_eager_model_version_query_options())
-                .filter(SqlModelVersion.current_stage != STAGE_DELETED_INTERNAL)
                 .order_by(*parsed_orderby)
                 .limit(max_results_for_query)
             )
