--- .venv/lib/python3.10/site-packages/mlflow/utils/search_utils.py	2023-07-17 10:46:19.499886698 +0200
+++ -	2023-07-17 10:47:07.422442839 +0200
@@ -1257,7 +1257,7 @@
     @classmethod
     def filter(cls, model_versions, filter_string):  # pylint: disable=arguments-renamed
         """Filters a set of model versions based on a search filter string."""
-        model_versions = [mv for mv in model_versions if mv.current_stage != STAGE_DELETED_INTERNAL]
+        model_versions = [mv for mv in model_versions]
         if not filter_string:
             return model_versions
         parsed = cls.parse_search_filter(filter_string)
